pyproject.toml

```diff
@@ -134,7 +134,7 @@ build-backend = "poetry.core.masonry.api"
 
     [tool.poe.tasks.test-docker-deps]
     help = "Initializes needed docker containers for tests"
-    cmd = "docker compose up -d grafana-test redis postgresql-test --wait"
+    cmd = "docker compose -f docker-compose.yaml -f docker-compose.test.yaml up -d grafana redis postgresql --wait"
```

Comment:
Just out of curiosity: Do we have to specify `grafana redis postgres` anymore if we always want to start everything of the docker-compose file? I'm just interested 😇